### ISSUE TITLE
Mac OS install error "damaged and can't be opened"

### ISSUE THREAD

**Issue by eshanrh** [ShinjiruApp/Shinjiru#40]
On mac os, a fresh install of Shinjiru from the latest release(3.3.4), the system throws an error of "Shinjiru.app is damaged and can't be opened. You should move it to the Trash". 

Surprisingly, the usual trick of clearing all attributes on the app with `xattr -cr Shinjiru.app` makes everything work fine for me.

I can't think of a reason for this, is it possible to fix by default? I'm not sure if it's even replicable.

mac os: 10.14.3 (Mojave)

**Comment by Kazakuri** [comment]
Sorry for the delay, I was at an anime convention this past weekend.

It seems like it's an issue with the code signing according to [this question](https://stackoverflow.com/questions/22458027/mac-dmg-oddity-signing-and-damaged-applications) on Stack Overflow.
And I can verify it locally:

```
$ codesign --verify --verbose=4 /Applications/Shinjiru.app
/Applications/Shinjiru.app: invalid signature (code or signature have been modified)
In architecture: x86_64
```

I'll try to play around with it to make sure it's getting signed properly, thanks for letting me know!

**Comment by Kazakuri** [comment]
Just released 3.4.0 which should be signed properly.
